Formula/slacknimate.rb

```diff
@@ -20,14 +20,10 @@ class Slacknimate < Formula
   depends_on "go" => :build
 
   def install
-    ENV["GOPATH"] = buildpath
-    pkgpath = buildpath/"src/github.com/mroth"
-    pkgpath.install Dir["*"]
-    cd pkgpath do
-      system "make"
-      bin.install "bin/slacknimate"
-      prefix.install_metafiles
-    end
+    system "go", "build",
+      "-ldflags", "-s -w -X main.version=#{version}",
+      "-trimpath", "-o", bin/"slacknimate", "./cmd/slacknimate"
+    prefix.install_metafiles
```

Comment:
I believe this would be good enough
```suggestion
    system "go", "build", "-X main.version=#{version}", *std_go_args, "./cmd/slacknimate"
```

Reply:
Ah cool, I wasn't aware of `std_go_args`. This would still need `-ldflags` as `-X foo` is actually an argument to ldflags, not its own top-level flag. I'll push a new commit with that change in a sec.

From looking at https://github.com/Homebrew/brew/pull/6938, `-s -w` ldflags are not included by default since there are some packages it might break, however in this case as I already have to pass ldflags to override the version, I'd like to include them to reduce binary size and try to be consistent with the binaries I provide for manual download via GitHub Releases. (LMK if this is an issue!)